src/ChairmarksExtras.jl

```diff
@@ -25,7 +25,7 @@ integer if the allocation pattern of `f` is nondeterministic.
 
 Similar to `(@b args...).bytes`.
 
-See `@be` for more info.
+See also [`Chairmarks.@be`](@ref).
```

Comment:
Fully qualifying `Chairmarks.@be` implies that `@be` is not exported; when in reality it is exported

Reply:
The reason why I was doing this is to distinguish between this and the text in the original docs:
```julia
Similar to `Base.@time`
```

Reply:
In a package called `ChairmarksExtras`, the exports of `Chairmarks` are perhaps more canonical than the exports of `Base`. Hence the qualification on `Base.@time` to distinguish.